File: 1-Iniciante/ex3214.py
#entrada
e, f, c = str(input()).split()
e = int(e)
f = int(f)
c = int(c)

#processamento
totalGarrafasVazias = e + f
totalRefri = 0

while totalGarrafasVazias >= c:
    aux = totalGarrafasVazias // c
    totalRefri += aux
    totalGarrafasVazias = aux + (totalGarrafasVazias % c)

#saida
print(totalRefri)
 

File: 1-Iniciante/Ex3214.java
import java.util.Scanner;

public class Ex3214 { //para ser aceito no URI mude o nome da classe para Main
    public static void main(String[] args) {
        // variaveis
        String[] valoresSeparados;
        int e, f, c, totalRefri = 0, totalGarrafasVazias;

        // entrada
        valoresSeparados = leitura();
        e = Integer.parseInt(valoresSeparados[0]);
        f = Integer.parseInt(valoresSeparados[1]);
        c = Integer.parseInt(valoresSeparados[2]);

        // processamento
        totalGarrafasVazias = e + f;
        while(totalGarrafasVazias >= c){
            int aux = totalGarrafasVazias / c;
            totalRefri += aux;
            totalGarrafasVazias = aux + (totalGarrafasVazias % c);
        }
        
        //saida
        System.out.println(totalRefri);
    }

    static String[] leitura() {
        Scanner scanner = new Scanner(System.in);
        String valores;

        valores = scanner.nextLine();
        scanner.close();
        return valores.split("\\s");
    }
}